content/docs/csidriver/features/powermax.md

```diff
@@ -611,4 +611,16 @@ vSphere:
 >Note: Replication is not supported with this feature.
 >Limitations of RDM can be referred [here.](https://configmax.esp.vmware.com/home)  
 >Supported number of RDM Volumes per VM is 60 as per the limitations.
->RDMs should not be added/removed manually from vCenter on any of the cluster VMs.   
+>RDMs should not be added/removed manually from vCenter on any of the cluster VMs.
+
+## Storage Capacity Tracking
+
+CSI PowerMax driver version 2.8.0 and above supports Storage Capacity Tracking.
+
+This feature helps the scheduler to make more informed choices about where to start pods which depend on unbound volumes with late binding (aka “wait for first consumer”). Nodes with requested capacity that is present on the storage array, satisfying the topology constraints will be available for scheduling the pods, otherwise the pods stay in pending state.
```

Comment:
Change sentence to read: " Nodes satisfying the topology constraints, and with requested capacity that is present on the storage array, will be available for scheduling the pods, Otherwise the pods stay in pending state."

Reply:
Changes made accordingly.